components/Modules/components/ModuleResourcesSection.tsx

```
@@ -0,0 +1,109 @@
+                  setResources(
+                    resources.map((r) =>
+                      r.id === updatedResource.id ? updatedResource : r
+                    )
+                  );
+                }
+              }}
+              showModuleColumn={false}
+            />
+          )
+            )}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)